include/utility.h

```diff
@@ -247,18 +250,19 @@ class ParamServer
     sensor_msgs::Imu imuConverter(const sensor_msgs::Imu& imu_in)
     {
         sensor_msgs::Imu imu_out = imu_in;
-        // rotate acceleration
-        Eigen::Vector3d acc(imu_in.linear_acceleration.x, imu_in.linear_acceleration.y, imu_in.linear_acceleration.z);
-        acc = extRot * acc;
-        imu_out.linear_acceleration.x = acc.x();
-        imu_out.linear_acceleration.y = acc.y();
-        imu_out.linear_acceleration.z = acc.z();
         // rotate gyroscope
         Eigen::Vector3d gyr(imu_in.angular_velocity.x, imu_in.angular_velocity.y, imu_in.angular_velocity.z);
         gyr = extRot * gyr;
         imu_out.angular_velocity.x = gyr.x();
         imu_out.angular_velocity.y = gyr.y();
         imu_out.angular_velocity.z = gyr.z();
+        // rotate acceleration
+        Eigen::Vector3d acc(imu_in.linear_acceleration.x, imu_in.linear_acceleration.y, imu_in.linear_acceleration.z);
+        acc = extRot * acc;
```

Comment:
I don't believe so because I am pre-multiplying here, which is equivalent to a post-multiplication in the opposite direction
```c++
acc = extRot * acc
// is equivalent to
acc = acc * inv(extRot)
```

Reply:
actually that entirely depends on whether the provided extrinsics matrix standands for the transformation of imu to lidar frame or lidar frame to imu frame. 

https://github.com/TixiaoShan/LIO-SAM/blob/9991daa47eb76124659570daab299bf877dde8bb/config/params.yaml#L41